docs/ssh-keys/link-ssh-key.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Link an SSH Key to a Server
+intro: Link an SSH Key to a server to select which SSH keys a server can have deployed.
+links:
+    overview:
+    quickstart:
+    previous:
+        - /docs/ssh-keys/add-ssh-key
+    next:
+        - /docs/ssh-keys/deploy-ssh-key
+    guides:
+    related:
+    featured:
+---
+
+1. On Devopness, navigate to a project then select an environment
+1. Find the `SSH Keys` card
+1. Click `View` in the `SSH Keys` card to see a list of existing `SSH Keys`
+1. Click `DETAILS` on the SSH Key you want to link to a server
+1. On the upper-right corner of the application details view, click `...`
```

Comment:
I suggest use lower case to 'key' when you are not referring to a button or label. Do you agree?

```suggestion
1. Click `DETAILS` on the SSH key you want to link to a server
1. On the upper-right corner of the SSH key details view, click `...`
```